src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -123,9 +123,18 @@ private theorem lt_two_pow_of_le {x m n : Nat} (lt : x < 2 ^ m) (le : m ≤ n) :
 
 /-! ### msb -/
 
-theorem msb_eq_decide (x : BitVec (Nat.succ w)) : BitVec.msb x = decide (2 ^ w ≤ x.toNat) := by
-  simp only [BitVec.msb, getMsb, Nat.zero_lt_succ,
-    decide_True, getLsb, Nat.testBit, Nat.succ_sub_succ_eq_sub,
+@[simp]
```

Comment:
I'm sorry if I'm contradicting myself, but I'd prefer this not be a simp lemma (for now). Just add it to the simp call in `msb_eq_decide`.